Sekoia.io/sekoiaio/triggers/intelligence.py

```diff
@@ -32,18 +35,22 @@ def feed_id(self) -> str:
     @property
     def url(self):
         url = (
-            urljoin(self.module.configuration["base_url"], f"v2/inthreat/collections/{self.feed_id}/objects")
+            urljoin(
+                self.module.configuration["base_url"],
+                f"v2/inthreat/collections/{self.feed_id}/objects",
+            )
             + f"?limit={self.__class__.batch_size_limit}"
         )
         if len(self.__class__.API_URL_ADDITIONAL_PARAMETERS) > 0:
             url += "&" + "&".join(self.__class__.API_URL_ADDITIONAL_PARAMETERS)
 
+        # depending on context, add cursor info and request revoked IOC
         with self.context as cache:
             cursor = cache.get("cursors", {}).get(self.feed_id)
             if cursor:
-                return f"{url}&cursor={cursor}"
+                return f"{url}&cursor={cursor}&include_revoked=true"
```

Comment:
This will include the revoked indicator not after the first full sync but after we fetch the first page.